src/main/java/com/mongodb/jdbc/MySQLDatabaseMetaData.java

```diff
@@ -907,6 +907,80 @@ public ResultSet getClientInfoProperties() throws SQLException {
         return new MySQLResultSet(null, new MongoExplicitCursor(docs), true);
     }
 
+    private MongoResultDoc getFunctionMetaDoc() {
+        MongoResultDoc metaDoc = new MongoResultDoc();
+        metaDoc.columns = new ArrayList<>(5);
```

Comment:
```suggestion
        metaDoc.columns = new ArrayList<>(6);
```
off-by-one?

Reply:
Ah yea, this has been "wrong" for over a year!